test/contracts/src/RelayEncoderInstance.sol

```diff
@@ -59,9 +58,12 @@ contract RelayEncoderInstance is RelayEncoder {
         return "0x00";
     }
 
-    function encodeSetController(
-        uint256 controller
-    ) external pure override returns (bytes memory result) {
+    function encodeSetController()
+        external
+        pure
+        override
+        returns (bytes memory result)
+    {
```

Comment:
quite weird that the contents of `RelayEncoder.sol` changed from this style to the one removed, but here it's the opposite

Reply:
You're right, I must have modified this file prior to switching the Solidity extensions that does the formatting. Is there a prefer way to format them?

Reply:
I think I just go with the vscode solidity extension and how it formats. Just strange that it formatted both files in 2 different ways. Maybe if you run the formatter again, it fixes them?

Reply:
Just tried formatting again and the result is the same. I think in this case it doesn't shrink to a single line because it would go over 80 chars

Reply:
xD okay then it's fine to leave it as is